config/config.msft.yaml

```diff
@@ -10,6 +10,10 @@ defaults:
     subscription: hcp-{{ .ctx.region }}
     region: uksouth
     globalMSIName: "global-ev2-identity"
+    parentZoneName: azure-test.net
+    baseDnsZoneName: aroapp-hcp.azure-test.net
+    regionalDNSSubdomain: '{{ .ctx.region }}'
+    svcParentZoneName: "aro-hcp.azure-test.net"
```

Comment:
- `aroapp-hcp.azure-test.net` exists under `dns.cxParentZoneName`
- `aro-hcp.azure-test.net` exists under `dns.svcParentZoneName`
- `regionalSubdomain` existsunder `dns.regionalSubdomain`
- please create `parentZoneName` under `dns.parentZoneName` (and declare it in `config.schema.json`)